grommet/grommet
# Update build script

#### What does this PR do?
Fixes build script to properly exclude test files as well as `portal.js`. Adds `copyfiles` as a dependency to replace `--copy-files` babel flag 
#### Where should the reviewer start?
`package.json` line 32, line 94
#### What testing has been done on this PR?
I have manually tested it by running yarn build and checking dist folder to confirm that the right files were excluded
#### How should this be manually tested?
Run yarn build and then browse through the grommet/dist directory to confirm that there are no test files included
#### What are the relevant issues?
#4118, #4119 related pull request [here](https://github.com/grommet/grommet/pull/4124)
#### Screenshots (if appropriate)
`/dist` folder before updating script
<img width="790" alt="Screen Shot 2020-05-28 at 3 52 13 PM" src="https://user-images.githubusercontent.com/49867133/83555906-06617780-a4dd-11ea-8bf9-0e6a5ebdf9a0.png">
`/dist/utils` folder before updating script
<img width="780" alt="Screen Shot 2020-05-28 at 3 52 35 PM" src="https://user-images.githubusercontent.com/49867133/83555922-0a8d9500-a4dd-11ea-9c9a-9fc7f7e20819.png">
`/dist/components/Box` folder before updating script (example of a component folder)
<img width="785" alt="Screen Shot 2020-05-28 at 3 52 50 PM" src="https://user-images.githubusercontent.com/49867133/83555931-0e211c00-a4dd-11ea-93e7-a6e60dec358f.png">
`/dist` folder after updating script
<img width="788" alt="Screen Shot 2020-06-02 at 2 28 20 PM" src="https://user-images.githubusercontent.com/49867133/83556434-e2eafc80-a4dd-11ea-864c-135c174ac2dd.png">
`/dist/utils` folder after updating script
<img width="781" alt="Screen Shot 2020-06-02 at 2 31 38 PM" src="https://user-images.githubusercontent.com/49867133/83556443-e67e8380-a4dd-11ea-9f30-eaec262523c2.png">
`/dist/components/Box` folder after updating script (example component)
<img width="782" alt="Screen Shot 2020-06-02 at 2 31 54 PM" src="https://user-images.githubusercontent.com/49867133/83556453-eb433780-a4dd-11ea-89b8-5cd501b3545a.png">


#### Do the grommet docs need to be updated?
No
#### Should this PR be mentioned in the release notes?
No
#### Is this change backwards compatible or is it a breaking change?
backwards compatible
